docs/guide/index.asciidoc

```diff
@@ -56,6 +56,48 @@ To visualize the data after it's sent to Elasticsearch,
 you can use the the dedicated APM UI bundled in X-Pack,
 or the pre-built open source Kibana dashboards that can be loaded directly in the Kibana UI.
 
+[[apm-release-notes]]
+=== Release notes
+
+[float]
+==== APM version 6.4
+
+[float]
+===== Breaking changes
+
+We previously split APM data into separate indices (transaction, span, error, etc.).
+In 6.4 APM Kibana UI starts to leverage those separate indices for queries.
+In case you only update Kibana but run an older version of APM Server you will not be able to see any APM data by default.
+To fix this, use the Kibana APM configuration options to specify the location of the APM index.
```

Comment:
Could you link to https://github.com/elastic/kibana/blob/2ad265b73f47efe2cfc5d209d8a4f006f3e5ca09/docs/settings/apm-settings.asciidoc?

Reply:
done here eaf82ead415fdc8eb574529899f43899f4f419eb